pkg/util/vsphere/vm/vm.go

```diff
@@ -22,3 +25,18 @@ func ManagedObjectFromMoRef(moRef vimtypes.ManagedObjectReference) mo.VirtualMac
 func ManagedObjectFromObject(obj *object.VirtualMachine) mo.VirtualMachine {
 	return ManagedObjectFromMoRef(obj.Reference())
 }
+
+func IsPausedByAdmin(moVM *mo.VirtualMachine) bool {
+	for i := range moVM.Config.ExtraConfig {
+		if o := moVM.Config.ExtraConfig[i].GetOptionValue(); o != nil {
+			// Break the loop when EC key is found.
+			if o.Key == vmopv1.PauseVMExtraConfigKey {
+				if o.Value == constants.ExtraConfigTrue {
+					return true
+				}
+				break
```

Comment:
nit: you could also do `return false` here as well.  And remove the comment on L32.

Reply:
It looks like the EC value is expected to be case sensitive?  IOW, we expect admins to specify "TRUE".  Anything else ("true, "True") won't work.  Is that documented somewhere in the API?  If not, we should call that out here: https://github.com/vmware-tanzu/vm-operator/blob/main/api/v1alpha3/virtualmachine_types.go#L154